packages/cli/src/Studio.ts

```diff
@@ -77,9 +77,9 @@ ${bold('Examples')}
    * Parses arguments passed to this command, and starts Studio
    *
    * @param argv Array of all arguments
-   * @param _config The loaded Prisma config
+   * @param config The loaded Prisma config
    */
-  public async parse(argv: string[], config: PrismaConfigInternal): Promise<string | Error> {
+  public async parse(argv: string[], config: PrismaConfigInternal<any>): Promise<string | Error> {
```

Comment:
ℹ️ This is now the only remaining use of `any` for the config stuff. I think we also cannot really avoid it here, or? 🤔

Reply:
Exactly. I think I also tried `unknown` here but it didn't let me :/